packages/mongodb-chatbot-ui/src/ChatWindow.tsx

```diff
@@ -134,6 +135,19 @@ export function ChatWindow(props: ChatWindowProps) {
                 {disclaimer}
               </DisclaimerText>
             ) : null}
+            <LGMessage
+              messageBody="Welcome to MongoDB AI. How can I help you today?"
+              isSender={false}
+              sourceType="markdown"
+              align="left"
+              links={[
+                {
+                  href: "https://www.mongodb.com",
+                  children: "MongoDB",
+                  variant: "Docs",
```

Comment:
should this be variant `Docs`? i'd think it's just general website

Reply:
Just put this here for debugging and forgot to remove